packages/legacy/core/App/constants.ts

```diff
@@ -63,6 +68,13 @@ export const attemptLockoutThresholdRules = {
 
 export const defaultAutoLockTime = 5
 
+export const tours: Tours = {
+  homeTourSteps: homeTourSteps,
+  credentialsTourSteps: credentialsTourSteps,
+  credentialOfferTourSteps: credentialOfferTourSteps,
+  proofRequestTourSteps: proofRequestTourSteps,
```

Comment:
Can be shortened to:
```suggestion
  homeTourSteps,
  credentialsTourSteps,
  credentialOfferTourSteps,
  proofRequestTourSteps,
```
Just if you want, you don't have to make this change. This way just saves the reader having to check that they are the same name and there's no slight difference